contents/projectManagement/githubActions.md

```diff
@@ -0,0 +1,142 @@
+
+
+![](images/marketplace.png)
+
+
+
+### 3. Cross OS Support
+GitHub Actions gives you the option to [run workflows](https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources) on Ubuntu, Windows and macOS.
+
+### 4. Docker Support
+GitHub Actions provides the ability to run docker images for projects that use a dockerized environment.
+
+### 5. Powerful
+Integration with the GitHub ecosystem allows GitHub Actions to do more than just build/test/deploy. You can automate everything related to your GitHub project, including bug triaging, labeling, automated dependency updating etc.
+Workflows can be triggered by events that happen on GitHub and will have access to the context of the issue/PR/commit that triggered it. 
+
+### 6. Affordable
+Like many of its competitors, GitHub Actions provides **free unlimited runner time for public repositories**. 
+
+For private repositories, the runner time you get with a free plan is limited, but still ample for hobby projects.
+
+---
+
+## Concepts
+<!--This is a simple high-level overview of the tool to give the reader some concrete sense of X (as opposed to limiting to an entirely abstract description). It's useful to give concrete examples such as code examples. Do not try to 'teach' how to use the X (assuming your in the style of a tutorial. If the tool is worth learning, there must be good tutorials about it already.-->
+
+* **Workflow**
+    A workflow describes a sequence of jobs to be run when a trigger event occurs. 
+    Each job consists of a series of steps, which may be actions or shell commands. 
+    Each workflow is defined as a `.yml` file in the `.github/workflows` directory of a repository.
+
+* **Action**
+    An action is a piece of code that performs an atomic task. 
+    For example, [checking out a repository](https://github.com/actions/checkout), or [caching dependencies](https://github.com/actions/cache).
+
+* **Triggers**
+    Workflows can be triggered by events on the GitHub repository, ranging from opening pull requests or issues, making commits to scheduled events, or even from an external event by calling GitHub's REST API endpoint.
+
+---
+
+## Set Up a Simple Workflow for a Node.js App
+<!--
+Provide a learning path for the reader. Try to give one good learning path rather than many random resources.
+--> 
+
+GitHub has made it extremely easy to set up a simple workflow without even leaving the browser. 
+
+![](images/actions-tab.png)
+
+1. Head to the Actions tab on your repository
+1. You can choose to set up a workflow from a template or from scratch. We will use the Node.js template in this example
+1. The online editor is powerful and easy to use, boasting features such as inline documentation and autocomplete
+![](images/editor.png)
+1. The template workflow will
+    1. Begin when a commit is pushed or a pull request is opened to `master`
```

Comment:
think can use a different type of ordered list aka (`i`, `ii`) here

Reply:
I prefer numbered sublists